exasol_script_languages_developer_sandbox/lib/run_reset_password.py

```diff
@@ -0,0 +1,30 @@
+from typing import Tuple
+
+from exasol_script_languages_developer_sandbox.lib.ansible.ansible_access import AnsibleAccess
+from exasol_script_languages_developer_sandbox.lib.ansible.ansible_context_manager import AnsibleContextManager
+from exasol_script_languages_developer_sandbox.lib.ansible.ansible_repository import AnsibleRepository, \
+    default_repositories
+from exasol_script_languages_developer_sandbox.lib.ansible.ansible_run_context import AnsibleRunContext, \
+    reset_password_ansible_run_context
+
+from exasol_script_languages_developer_sandbox.lib.host_info import HostInfo
+
+
+def run_reset_password(ansible_access: AnsibleAccess, default_password: str,
+                       host_infos: Tuple[HostInfo, ...] = tuple(),
+                       ansible_run_context: AnsibleRunContext = reset_password_ansible_run_context,
+                       ansible_repositories: Tuple[AnsibleRepository, ...] = default_repositories) -> None:
+    """
+    Resets the password and removes the .ssh directory via ansible.
+    The ansible working dir is created dynamically and removed afterwards.
+    Any hosts, given by variable host_infos, are added to the ansible inventory in the dynamic working directory.
```

Comment:
use param docstrings

Reply:
https://sphinx-rtd-tutorial.readthedocs.io/en/latest/docstrings.html

Reply:
Hm, I think this is a thing we need to discuss. Until now I tried to explain the parameters in the function description. (I think this is the way Google does it, too).
If we agree to use the param docstring, I will need to change a lot of function documentation

Reply:
I created #20 to taggle this one.